tests/test_transforms/test_math/test_binary_operator_transform.py

```diff
@@ -0,0 +1,256 @@
+        (">=", "target", "feature", "target"),
+        ("<=", "target", "feature", "target"),
+        (">", "target", "feature", "target"),
+        ("<", "target", "feature", "target"),
+        ("+", "feature", "target", "new_col"),
+        ("-", "feature", "target", "new_col"),
+        ("*", "feature", "target", "new_col"),
+        ("/", "feature", "target", "new_col"),
+        ("/=", "feature", "target", "new_col"),
+        ("%", "feature", "target", "new_col"),
+        ("**", "feature", "target", "new_col"),
+        ("==", "feature", "target", "new_col"),
+        (">=", "feature", "target", "new_col"),
+        ("<=", "feature", "target", "new_col"),
+        (">", "feature", "target", "new_col"),
+        ("<", "feature", "target", "new_col"),
+        ("+", "target", "feature", "new_col"),
+        ("-", "target", "feature", "new_col"),
+        ("*", "target", "feature", "new_col"),
+        ("/", "target", "feature", "new_col"),
+        ("/=", "target", "feature", "new_col"),
+        ("%", "target", "feature", "new_col"),
+        ("**", "target", "feature", "new_col"),
+        ("==", "target", "feature", "new_col"),
+        (">=", "target", "feature", "new_col"),
+        ("<=", "target", "feature", "new_col"),
+        (">", "target", "feature", "new_col"),
+        ("<", "target", "feature", "new_col"),
+    ],
+)
+def test_simple_two_segments(ts_two_segments: TSDataset, operand, left_column, right_column, out_column):
+    left_vals1 = deepcopy(ts_two_segments.df["segment_0"][left_column].values)
+    right_vals1 = deepcopy(ts_two_segments.df["segment_0"][right_column].values)
+    left_vals2 = deepcopy(ts_two_segments.df["segment_1"][left_column].values)
+    right_vals2 = deepcopy(ts_two_segments.df["segment_1"][right_column].values)
+    checker_vals1 = ops[operand](left_vals1, right_vals1)
+    checker_vals2 = ops[operand](left_vals2, right_vals2)
+    transformer = binary_operator.BinaryOperationTransform(
+        left_column=left_column, right_column=right_column, operator=operand, out_column=out_column
+    )
+    new_ts = transformer.fit_transform(ts=ts_two_segments)
+    new_ts_vals1 = new_ts.df["segment_0"][out_column].to_numpy()
+    new_ts_vals2 = new_ts.df["segment_1"][out_column].to_numpy()
+    numpy.testing.assert_array_almost_equal(new_ts_vals1, checker_vals1)
+    numpy.testing.assert_array_almost_equal(new_ts_vals2, checker_vals2)
+
+
+@pytest.mark.parametrize(
+    "operand, left_column, right_column, out_column",
+    [
+        ("+", "feature", "target", "target"),
+        ("-", "feature", "target", "target"),
+        ("*", "feature", "target", "target"),
+        ("/", "feature", "target", "target"),
+        ("+", "target", "feature", "target"),
+        ("-", "target", "feature", "target"),
+        ("*", "target", "feature", "target"),
+        ("/", "target", "feature", "target"),
+    ],
+)
+def test_inverse_one_segment(ts_one_segment, operand, left_column, right_column, out_column):
```

Comment:
We should add situations like "target", "feature", "feature"

Reply:
Do we? Columns `target` and `feature` are almost the same (by distribution) how is it different from "feature", "target", "target"?